BEIMA.Backend/MongoService/BaseDevice.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace BEIMA.Backend.MongoService
+{
+    /// <summary>
+    /// This class represents a Device. This object contains all the required fields necessary
+    /// for a device. This is meant to be used to convert data received from an endpoint, back into a BSON object.
+    /// </summary>
+    public class BaseDevice
+    {
+
+        /// <summary>
+        /// Empty constructor, this allows for a device to be instantiated through Object Initializers.
+        /// </summary>
+        public BaseDevice()
+        {
+            Fields = new BsonDocument();
+            Location = new BsonDocument();
+            LastModified = new BsonDocument();
+        }
+
+        /// <summary>
+        /// Full constructor to instantiate a Device object.
+        /// </summary>
+        /// <param name="id">ObjectId of the object itself.</param>
+        /// <param name="deviceTypeId">ObjectId of the DeviceType this Device is linked to.</param>
+        /// <param name="deviceTag"></param>
+        /// <param name="manufacturer"></param>
+        /// <param name="modelNum"></param>
+        /// <param name="serialNum"></param>
+        /// <param name="yearManufactured"></param>
+        /// <param name="notes"></param>
```

Comment:
params

Reply:
I thought these were self explanatory, what type of param comments need to be put here?

Reply:
You can literally just copy the param name if it's simple enough. Like "The device tag" for "deviceTag".

Reply:
Resolved [95e7484](https://github.com/cs481-ekh/s22-beima/pull/114/commits/95e7484f1c1e7fa2a0b2e1fa32c92df232cf3a0d)